language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/util/SortingUtil.java

```diff
@@ -215,11 +218,54 @@ public static String genSortTextForTypeDescContext(BallerinaCompletionContext co
         if (item.getType() == SNIPPET && ((SnippetCompletionItem) item).kind() == SnippetBlock.Kind.TYPE) {
             return genSortText(8);
         }
-        
+
         // Case 7+ 
         return genSortText(SortingUtil.toRank(item, 8));
     }
 
+    /**
+     * Generate the sort text based on the assignability to particular type symbol.
+     * <p>
+     * Sorting order is defined as follows
+     * 1.Variables assignable to the given type symbol
+     * 2.Functions with return type descriptor assignable to given type symbol
+     * 3.Other symbols follows the default sorting order
```

Comment:
```suggestion
     * 3.Other symbols follow the default sorting order
```
Shall we make this change?

Reply:
Done